assessment_module_manager/modules.docker.ini

```
@@ -32,3 +32,8 @@ supports_evaluation = false
 url = http://module-programming-code-embedding:5007
 type = programming
 supports_evaluation = false
+
+[module_modelling_llm]
+url = http://module-modelling-llm:5008
+type = modelling
+supports_evaluation = false
```

Shouldn't this be true? 🤔
```suggestion
supports_evaluation = true
```